grc/gui_qt/properties.py

```diff
@@ -53,6 +53,7 @@ class Paths(object):
     TOP_BLOCK_FILE_MODE = HIER_BLOCK_FILE_MODE | stat.S_IXUSR | stat.S_IXGRP
 
     # Setup paths
+    # FIXME: this should use ..main.get_config_directory() and get_state_directory(), probably.
```

Comment:
@haakov  I wasn't sure what to do with the commented-out code. Do we get to keep it or should we clean it up?

Reply:
I kind of want to get rid of this file. We have Config.py, we have Constants.py, and we have Platform.py in the same directory, and I'm not quite sure what goes where. Maybe we can move the contents of this file to the others?

Reply:
OK, then let's do that, it's OK enough an effort to be done quickly; but let's keep that out of this PR, then. I guess that especially means that the commented out code on Lines 57ff will just be removed.